server/views/match/placementRequests/spaceBookings/new.njk

```diff
@@ -1,10 +1,18 @@
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
 
 {% extends "../../layout-with-details.njk" %}
 
 {% set pageTitle = applicationName + " - " + pageHeading %}
 
+{% block beforeContent %}
+  {{ govukBackLink({
+		text: "Back",
```

Comment:
Why not use `paths.v2Match.placementRequests.search.spaces({ id: placementRequest.id })` here?

Reply:
good point